### ISSUE TITLE
Needs to be updated to 0.0.4

### ISSUE THREAD

**Issue by iBurley** [flathub/com.discordapp.Discord#14]
Discord has updated once again, download link:
https://discordapp.com/api/download?platform=linux&format=tar.gz

**Comment by kparal** [comment]
Failed to build:
```
com.discordapp.Discord.desktop:    AppData problem: attribute-missing : <release> has no version
com.discordapp.Discord.desktop:    AppData problem: attribute-missing : <release> has no timestamp
Error loading AppData file: AppData file /app/share/appdata/com.discordapp.Discord.appdata.xml was not valid
Error: ERROR: appstream-compose failed: Child process exited with code 1
```
https://flathub.org/builds/#/builders/4/builds/1480

**Comment by kparal** [comment]
Seems to be fixed with #16, once merged.
